docs/components/rating/README.md

```diff
@@ -101,7 +101,7 @@
                   "type": "string",
                   "default": "medium",
                   "required": "false",
-                  "description": "Modifies the style variant for this component. Possible values: {  ‘small’  |  ‘medium’  |  ‘large’  }"
+                  "description": "Modifies the size variant for this component. Possible sizes: {  ‘small’  |  ‘medium’  |  ‘large’  }. Also works with responsive breakpoints. Example: { 'large@xs' }"
                 },
```

Comment:
Same here... I like how you have the size description in button:  ```"Sets the button size; values can target responsive breakpoints. Breakpoint values are: xs, sm, md, and lg. Examples: { 'small' | 'medium' | 'large' | 'large@sm' }"```
Maybe also change this to be consistent?